doc/rados/configuration/common.rst

```diff
@@ -7,65 +6,69 @@ Common Settings
 The `Hardware Recommendations`_ section provides some hardware guidelines for
 configuring a Ceph Storage Cluster. It is possible for a single :term:`Ceph
 Node` to run multiple daemons. For example, a single node with multiple drives
-may run one ``ceph-osd`` for each drive. Ideally, you will  have a node for a
-particular type of process. For example, some nodes may run ``ceph-osd``
-daemons, other nodes may run ``ceph-mds`` daemons, and still  other nodes may
-run ``ceph-mon`` daemons.
+can run one ``ceph-osd`` for each drive. Ideally, a particular node will be
+assigned to a particular type of process. For example, some nodes might run
+``ceph-osd`` daemons, other nodes might run ``ceph-mds`` daemons, and still
+other nodes might run ``ceph-mon`` daemons.
+
+Each node has a name. The name of a node can be found in its ``host`` setting.
+Monitors also specify a network address and port (that is, a domain name or IP
+address) that can be found in the ``addr`` setting. A basic configuration file
+typically specifies only minimal settings for each instance of monitor daemons.
+For example:
```

Comment:
@rzarzynski - I find it difficult to understand and revise this paragraph. I'm not sure if the topic of the paragraph is settings of a node, settings of a monitor daemon, or a variety of loosely related settings. I would appreciate any guidance on the paragraph as a whole, but (if it helps) here are some specific questions I have: (1) When the third sentence speaks of a "network address and port", are these supposed to be the network address and port of a node (just as the name is the name of a node)? (2) When monitors specify the network address and port, does that mean that the monitors are determining the value of the ``addr`` setting? (3) Is "basic configuration file" a common term of art here, or is there another term that is more common or more precise? (4) Can "each instance of monitor daemons" be replaced with "each monitor daemon"? (5) When the fourth sentence speaks of "minimal settings" for a monitor daemon (or for an instance of monitor daemons), are these also minimal settings of a node? Do these settings include the ``host`` and ``addr`` settings mentioned in the previous sentences?

Thanks for any help.

Reply:
unfinished business